tests/common/common.h

```diff
@@ -158,6 +158,22 @@ void check_get_info_param(const ObjectT& object) {
   auto returnValue = object.template get_info<InfoDesc>();
   check_return_type<ReturnT>(returnValue, "object::get_info()");
 }
+/**
```

Comment:
Let's keep existing style i.e. separate function templates by empty line.
```suggestion
}

/**
```